tests/integration_tests/test_applications_from_config.py

```diff
@@ -294,6 +296,8 @@ def prepare_configuration(config, output_path):
         return None, "--" + list(config.keys())[0].lower()
 
     tmp_config_file = output_path / "tmp_config.yml"
+    if model_version is not None and "MODEL_VERSION" in config:
+        config.update({"MODEL_VERSION": model_version})
```

Comment:
I am not entirely sure these lines are doing what we expect. We prioritize here the parameter passed over the one in the config, but perhaps we gave the version in the config file because that is the one we expect it to work with, no?

Reply:
This replaces the `model_version` for all tools which have a `model_version` set via the config file. 

This is exactly what we want to do and allow us to run the integration test using a command line parameter (e.g., `--model_version=prod5` to run all integration tests with prod6.

Reply:
Ok, so that means all integration tests should work with both versions? For instance, we don't have integration tests that should work only with prod5 ?

Reply:
All tests should work with both versions (and they do). I do think we should aim that this is also true in the future.

The only difference we have to make if we check for output files. At this point, we only check for the version which is labeled in the configuration files (file naming is a bit complicated and adding this is probably not necessary)